src/cluecode/finder.py

```diff
@@ -59,6 +59,23 @@ def logger_debug(*args):
 """
 
 
+def check_if_line_can_contain_urls(line):
+    """
+    Returns true if the line is a likely candidate to include urls.
+    """
+    url_keyword_list = ['://', 'http', 'https', 'www', 'git', 'ftp', 'ftps',
+                        'sftp', 'rsync', 'ssh', 'svn', 'hg']
+    return any(item in line for item in url_keyword_list)
+
+
+def check_if_line_can_contain_emails(line):
+    """
+    Returns true if the line is a likely candidate to include emails.
+    """
+    email_keyword_list = ['@']
+    return any(item in line for item in email_keyword_list)
```

Comment:
I am not sure why you are doing it this way. What about a simpler `return '@' in line` ?

Reply:
I thought of taking care of items that would be added to the list in future so....
Nevermind, I'll change it 👍

Reply:
Addressed your comment. :)